docker-compose.ccm.yaml

```diff
@@ -39,6 +39,20 @@ services:
       - traefik.http.middlewares.app.headers.accessControlAllowCredentials=true
       - traefik.http.middlewares.app.headers.frameDeny=true
       - traefik.http.middlewares.app.headers.browserXssFilter=true
+  app-muise:
+    <<: *app
+    environment:
+      <<: *env
+      ST_SECRET_KEY: "${MUISE_ST_SECRET_KEY}"
+      ST_DATABASE_URI: "mysql+pymysql://${MUISE_MYSQL_CONNECTION_STRING}"
+      MINIO_ENDPOINT: "${MUISE_MINIO_ENDPOINT}"
+      MINIO_ACCESS_KEY: "${MUISE_MINIO_ACCESS_KEY}"
+      MINIO_SECRET_KEY: "${MUISE_MINIO_SECRET_KEY}"
+    labels:
+      - traefik.enable=true
+      - traefik.http.routers.app-hawkins.rule=Host(`stager-muise.ccm.sickkids.ca`) # && PathPrefix(`/api`)
```

Comment:
For these 3 lines, I think you meant to put `app-muise` as opposed to `app-hawkins`.

Reply:
Oops, good catch.